opensubtitles/infos.go

```diff
@@ -0,0 +1,91 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+// InfoService provides access to the info related functions
+// in the OpenSubtitles API.
+//
+// OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#infos
+type InfoService service
+
+type UserData struct {
```

Comment:
Codacy found an issue: [exported type UserData should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)